sdks/java/io/pom.xml

```diff
@@ -67,6 +67,7 @@
     <module>elasticsearch</module>
     <module>google-cloud-platform</module>
     <module>hadoop-common</module>
+	<module>hadoop-input-format</module>
```

Comment:
Could you please let us know what formatting are you exactly talking about? Is the name format incorrect?